packages/gatsby/package.json

```diff
@@ -145,7 +145,10 @@
     "webpack-merge": "^4.2.2",
     "webpack-stats-plugin": "^0.3.1",
     "xstate": "^4.8.0",
-    "yaml-loader": "^0.5.0"
+    "yaml-loader": "^0.5.0",
+    "zipkin": "^0.19.2",
+    "zipkin-transport-http": "^0.19.2",
+    "zipkin-javascript-opentracing": "^2.1.0"
```

Comment:
I think we need to move these to dev deps

Reply:
You’re right. I moved  Those  to devs.

https://github.com/gatsbyjs/gatsby/pull/22372/commits/7d68a33a8f1c4eb84f7ca975ece1944239829b18